packages/node_modules/@webex/internal-plugin-dss/src/index.ts

```diff
@@ -0,0 +1,8 @@
+import '@webex/internal-plugin-mercury';
```

Comment:
should we move the mercury require into the dss.ts file , If at some point we need to move the business logic its easy

Reply:
done